### ISSUE TITLE
span styles inside heading are not displayed in CP  

### ISSUE THREAD

**Issue by dadaxr** [jacksleight/statamic-bard-texstyle#62]
## Bug Description
When applying a "span" style on a word inside an heading (h1-h6), ... the markup is correctly updated (ie: creating a new span node, with the correct class to it) but the styles does not show in the control panel... But it works on the frontend...

The markup : 

![image](https://github.com/jacksleight/statamic-bard-texstyle/assets/1267495/bbb12443-d7ff-4dff-a0e5-3ce9c6372d23)

The preview : 

![image](https://github.com/jacksleight/statamic-bard-texstyle/assets/1267495/39f9c813-5c1a-4488-bb52-e72c0f02a738)

After a quick look, it's because the "selector" tied to the "span" type (in TypeManager.php) is :

```
'span' => [
            'display' => 'Span',
            'selector' => ':is(p, ul, ol, blockquote) span',
```   

If I change the selector with : 

```
'span' => [
            'display' => 'Span',
            'selector' => ':is(p, ul, ol, blockquote, h1, h2, h3, h4, h5, h6) span',
```

the styles are correctly shown on the control panel : 

![image](https://github.com/jacksleight/statamic-bard-texstyle/assets/1267495/06996ae5-8dad-4d4b-8555-2dc74d52f3c2)

From my experience, it's a common pattern to use span to style stuff, including inside titles. What do you think ?

**Comment by jacksleight** [comment]
Ah yep, good point. Will take a look.

**Comment by jacksleight** [comment]
Fixed in v3.1.5.
